### ISSUE TITLE
How to compile executable files with c4

### ISSUE THREAD

**Issue by MaDinnerbone** [rswier/c4#79]
i want to compile an executable file by c4 itself

**Comment by rswier** [comment]
Take a look at this fork of c4 by Dmytro Sirenko that includes a version that targets x86 execution.  Hopefully that helps!
https://github.com/EarlGray/c4

**Comment by EarlGray** [comment]
Yes, taking the generated machine code and dumping it into a simple ELF file should be straightforward.

**Comment by ljhhh0123** [comment]
It is compiled into intermediate code, then interpreted and executed, and cannot generate executable files.
If you want to generate executable programs, please refer to Fbrice Bellard's otccelfn.c
